modules/javafx.graphics/src/main/java/com/sun/prism/d3d/D3DVramPool.java

```diff
@@ -48,16 +48,15 @@ public long size(D3DTextureData resource) {
     public long estimateTextureSize(int width, int height,
                                     PixelFormat format)
     {
-        return ((long) width) * ((long) height) *
-               ((long) format.getBytesPerPixelUnit());
+        return (long) width * height * format.getBytesPerPixelUnit();
     }
 
     @Override
     public long estimateRTTextureSize(int width, int height,
                                       boolean hasDepth)
     {
         // REMIND: need to deal with size of depth buffer, etc.
-        return ((long) width) * ((long) height) * 4L;
+        return (long) width * height * 4;
```

Comment:
Ditto

Reply:
4L * w * h;

Reply:
I would just revert it back.

Reply:
(long) width * height * 4;
is dangerous if one does not remember the precedence rules - is typecast applied to width or the end result?
4L * w * h; is unambiguous.

Reply:
I'm not sure I like either of the two choices you listed. By "revert it back" I mean leave the code alone. The existing code, with its explicit casts, leave no room for doubt:

```
return ((long) width) * ((long) height) * 4L;
```